app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
```

Comment:
The import statement is correct, but ensure that the `Callable` and `Any` types are used appropriately in the function signatures.